Lombiq.HelpfulLibraries.OrchardCore/ResourceManagement/IResourceFilterProvider.cs

```diff
@@ -15,6 +15,9 @@ public interface IResourceFilterProvider
 
 public static class ResourceFilterProviderExtensions
 {
+    /// <summary>
+    /// Gets the required themes from the provider.
```

Comment:
What's a "required" theme?

Reply:
A theme that is required. A requirement which is theme. A theme requirement.

Philosophy aside, what might be the best way to describe this? How about `Gets the themes required by the <see cref="ResourceFilterThemeRequirementAttribute"/> from the provider.`?

Reply:
Yep, that's better.